bin/hammer

```diff
@@ -52,12 +52,16 @@ end
 
 CFG_PATH_LOCAL = ['./config/']
 
+HammerCLI::Settings.load_from_paths CFG_PATH_LOCAL
+
 if preparser.config
-  CFG_PATH_LOCAL.unshift preparser.config
+  if File.file? preparser.config
+    HammerCLI::Settings.load_from_file preparser.config
```

Comment:
The tilde shouldn't even get there. It should get expanded by the shell, unless the user escapes it, in which case I'd think the tilde is part of the path as a literal